packages/pg-connection-string/index.js

```diff
@@ -201,6 +206,20 @@ function parseIntoClientConfig(str) {
   return toClientConfig(parse(str))
 }
 
+function deprecatedSslModeWarning(sslmode) {
+  if (!deprecatedSslModeWarning.warned) {
+    deprecatedSslModeWarning.warned = true
+    emitWarning(`SECURITY WARNING: The SSL modes 'prefer', 'require', and 'verify-ca' are treated as aliases for 'verify-full'.
+In the next major version (v3.0.0), these modes will adopt standard libpq semantics, which have weaker security guarantees.
+
```

Comment:
nit: I think this would probably be considered a breaking change to `pg` itself so might want to call out we'd be bumping pg semver to 9.x at this point as well?

Reply:
called out both packages

Reply:
awesome! ty! will merge now. 😄